book/01-introduction/sections/what-is-git.asc

```diff
@@ -30,6 +30,7 @@ Git arbeitet im Großen und Ganzen eher wie ein mit einigen unglaublich mächtig
 ==== Fast jede Funktion arbeitet lokal
 
 Die meisten Aktionen in Git benötigen nur lokale Dateien und Ressourcen, um ausgeführt zu werden – im Allgemeinen werden keine Informationen von einem anderen Computer in Ihrem Netzwerk benötigt.
+// "diesen"? Bisher war von "ihm" noch nie die Rede. --> einen
```

Comment:
Das "you" und "your" wird öfter verwendet, um den Leser direkt anzusprechen. Ich sehe hier mit "einem" keine gute Alternative.

Reply:
Ok, du hast die nächste Zeile gemeint.

Reply:
Auch hier bezieht sich der Kommentar auf die nachfolgende Zeile (in der vorherigen gibt es ja auch gar kein "diesen"). Dort ist die Rede von 

> diesen Netzwerk-Latenz-Overhead

Allerdings ist vorher noch nie von einem Overhead geschrieben worden, d.h. er ist noch nicht eingeführt, weshalb man nicht "diesen" schreiben kann/sollte.